### PR TITLE
Mute audio for testing

### PR DESCRIPTION
**Pull request by tibetiroka**
**Feature:** This PR mutes the game when executing tests.

## Feature Details
I just added a quick `Audio::SetVolume(0)` if a test is passed to the game loop. This doesn't affect any default settings for the volume, nor does it change how the game behaves when not running tests.

## Testing Done
It muted during tests, and didn't mute when playing.

### CODE REVIEW

**Comment by tibetiroka** [comment]
It works when I use quyykk's ctest command, if that's what you mean

**Comment by warp-core** [comment]
Can you add support for an additional argument `--nomute` that would override this auto-muting?

**Comment by tibetiroka** [comment]
I guess I could add that. I'm not sure how I feel about adding a new argument just for this, but it doesn't sound too difficult.

File: endless-sky.6 [hunk 1]
```diff
@@ -39,11 +39,14 @@ sets the directory where preferences and saved games will be stored.
 prints any content or whitespace\-formatting errors found while loading data files and the most recent saved game. This option prevents the game from launching.
 
 .IP \fB\-\-test\ <name>
-execute the test case with the given name
+execute the test case with the given name.
```
Comment (warp-core) [inline endless-sky.6:42]:
```suggestion
execute the test case with the given name. By default, the game will be muted while running tests.
```

**Review by warp-core — APPROVED** [review]

**Pull request merged**